test/e2e/test_tools.py

```diff
@@ -29,62 +30,133 @@ def tearDown(self) -> None:
+        self.assertEqual(sorted(user_email_expected), sorted(user_email_received))
+        self.assertEqual(sorted(user_given_name_expected), sorted(user_given_name_received))
+        self.assertEqual(sorted(user_family_name_expected), sorted(user_family_name_received))
+        self.assertEqual(sorted(user_lang_expected), sorted(user_lang_received))
+
+        ontos_expected = test_onto['project']['ontologies']
+        ontos_received = test_onto_out['project']['ontologies']
+        onto_names_expected = []
+        onto_labels_expected =[]
+        onto_names_received = []
+        onto_labels_received = []
+        for onto in ontos_expected:
+            onto_names_expected.append(onto["name"])
+            onto_labels_expected.append(onto["label"])
+        for onto in ontos_received:
+            onto_names_received.append(onto["name"])
+            onto_labels_received.append(onto["label"])
+        self.assertEqual(sorted(onto_names_expected), sorted(onto_names_received))
+        self.assertEqual(sorted(onto_labels_expected), sorted(onto_labels_received))
+
+        test_list: Any = None
+        test_list_out: Any = None
         not_used_list: Any = None
         not_used_list_out: Any = None
-        tree_list_root: Any = None
-        tree_list_root_out: Any = None
+        excel_list: Any = None
+        excel_list_out: Any = None
 
-        for anything_list in anything_onto['project']['lists']:
-            list_name = anything_list.get('name')
-            if list_name == 'otherTreeList':
-                other_tree_list = anything_list
+        for test_onto_list in test_onto['project']['lists']:
+            list_name = test_onto_list.get('name')
+            if list_name == 'testlist':
+                test_list = test_onto_list
             elif list_name == 'notUsedList':
-                not_used_list = anything_list
-            elif list_name == 'treelistroot':
-                tree_list_root = anything_list
-
-        for anything_list in anything_onto_out['project']['lists']:
-            list_name = anything_list.get('name')
-            print(anything_list.get('name'))
-            if list_name == 'otherTreeList':
-                other_tree_list_out = anything_list
+                not_used_list = test_onto_list
+            elif list_name == 'my-list-from-excel':
+                excel_list = test_onto_list
+
+        for test_onto_list in test_onto_out['project']['lists']:
+            list_name = test_onto_list.get('name')
+            if list_name == 'testlist':
+                test_list_out = test_onto_list
             elif list_name == 'notUsedList':
-                not_used_list_out = anything_list
-            elif list_name == 'treelistroot':
-                tree_list_root_out = anything_list
+                not_used_list_out = test_onto_list
+            elif list_name == 'my-list-from-excel':
+                excel_list_out = test_onto_list
```

Comment:
this block is quite an eye sore... ;)

can I suggest the following:

```python

lists = anything_onto['project']['lists']
test_list = next((l for l in lists if l['name'] == 'testlist'), None)
not_used_list = next((l for l in lists if l['name'] == 'notUsedList'), None)
excel_list = next((l for l in lists if l['name'] == 'my-list-from-excel'), None)


lists_out = anything_onto_out['project']['lists']
test_list_out = next((l for l in lists_out if l['name'] == 'testlist'), None)
not_used_list_out = next((l for l in lists_out if l['name'] == 'notUsedList'), None)
excel_list_out = next((l for l in lists_out if l['name'] == 'my-list-from-excel'), None)

```

unless I'm overlooking something, it should do the exact same thing, and I think it's a bit easier on the eye

Reply:
Nice suggestion, thanks!